src/node_options.cc

```diff
@@ -699,76 +758,114 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::experimental_default_config_file);
   AddOption("--test",
             "launch test runner on startup",
-            &EnvironmentOptions::test_runner);
+            &EnvironmentOptions::test_runner,
+            kDisallowedInEnvvar);
   AddOption("--test-concurrency",
             "specify test runner concurrency",
-            &EnvironmentOptions::test_runner_concurrency);
+            &EnvironmentOptions::test_runner_concurrency,
+            kDisallowedInEnvvar,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--test-force-exit",
             "force test runner to exit upon completion",
-            &EnvironmentOptions::test_runner_force_exit);
+            &EnvironmentOptions::test_runner_force_exit,
+            kDisallowedInEnvvar,
+            false,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--test-timeout",
             "specify test runner timeout",
-            &EnvironmentOptions::test_runner_timeout);
+            &EnvironmentOptions::test_runner_timeout,
+            kDisallowedInEnvvar,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--test-update-snapshots",
             "regenerate test snapshots",
-            &EnvironmentOptions::test_runner_update_snapshots);
+            &EnvironmentOptions::test_runner_update_snapshots,
+            kDisallowedInEnvvar,
+            false,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--experimental-test-coverage",
             "enable code coverage in the test runner",
-            &EnvironmentOptions::test_runner_coverage);
+            &EnvironmentOptions::test_runner_coverage,
+            kDisallowedInEnvvar,
+            false,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--test-coverage-branches",
             "the branch coverage minimum threshold",
             &EnvironmentOptions::test_coverage_branches,
-            kAllowedInEnvvar);
+            kAllowedInEnvvar,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--test-coverage-functions",
             "the function coverage minimum threshold",
             &EnvironmentOptions::test_coverage_functions,
-            kAllowedInEnvvar);
+            kAllowedInEnvvar,
+            OptionNamespaces::kTestRunnerNamespace);
   AddOption("--test-coverage-lines",
             "the line coverage minimum threshold",
             &EnvironmentOptions::test_coverage_lines,
-            kAllowedInEnvvar);
-
-  AddOption("--experimental-test-isolation",
+            kAllowedInEnvvar,
+            OptionNamespaces::kTestRunnerNamespace);
+  AddOption("--test-isolation",
```

Comment:
I don't think we have consensus to remove the experimental status from test isolation in 22.x? I don't even know if all of the relevant commits are on v22.x as a lot of test runner related PRs are awaiting manual backports cc @nodejs/test_runner